packages/massa-web3/src/web3/SmartContractsClient.ts

```diff
@@ -147,7 +147,7 @@ export class SmartContractsClient
 
     if (callData.maxGas === null || callData.maxGas === undefined) {
       const reponse = await this.readSmartContract({
-        maxGas: BigInt(0),
+        maxGas: BigInt(2_100_000),
```

Comment:
Could you add a comment to explain why we need this value and maybe make a constant in web3-utils if that make sense